server/Makefile

```diff
@@ -291,7 +291,7 @@ else
 endif
 
 golangci-lint: ## Run golangci-lint on codebase
-	@# Keep the version in sync with the command in .circleci/config.yml
+	@# Keep the version in sync with the command in .github/workflows.yml#L99
```

Comment:
This file does not exist, what should it say instead?

Reply:
lol, it should say .github/workflows/ci.yml#L99

Reply:
But line 99 in that file is just `cache-dependency-path: server/go.sum` :joy:

Reply:
Yeah I was switching between branches. Got that messed up.